spyderlib/config.py

```diff
@@ -717,7 +743,7 @@ def is_ubuntu():
 # 2. If you want to *remove* options that are no longer needed in our codebase,
 #    you need to do a MAJOR update in version, e.g. from 3.0.0 to 4.0.0
 # 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '17.0.0'
+CONF_VERSION = '17.1.0'

```

Comment:
Please update this value to `18.1.0`. I merged PR #2237 with the wrong `CONF_VERSION` and now we need this value :-)